package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "delay",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump :)

Reply:
Haha, makes sense.  I only did it to address

> Do it in the next major version.